validator/schema.go

```diff
@@ -179,7 +179,7 @@ func validateDefinition(schema *Schema, def *Definition) *gqlerror.Error {
 		if err := validateArgs(schema, field.Arguments, nil); err != nil {
 			return err
 		}
-		if err := validateDirectives(schema, field.Directives, nil); err != nil {
+		if err := validateDirectives(schema, field.Directives, def.Kind, nil); err != nil {
```

Comment:
```suggestion
		if err := validateDirectives(schema, field.Directives, LocationFieldDefinition, nil); err != nil {
```
The directives here are on `FIELD_DEFINITION`, not on Type Definition.

Reply:
Done.